### PR TITLE
[CI] Set correct permissions for auto_request_review

### PR DESCRIPTION
**Pull request by malfet**

### CODE REVIEW

File: .github/workflows/auto_request_review.yml [hunk 1]
```diff
@@ -3,11 +3,14 @@ name: Auto Request Review
...
     types: [opened, ready_for_review, reopened]
-
+   
 jobs:
   auto-request-review:
     # Don't run on forked repos
     if: ${{ !github.event.pull_request.head.repo.fork }}
+    permissions:
+     contents: read
+      pull-requests: write
```
Comment (izaitsevfb) [inline .github/workflows/auto_request_review.yml:13]: Is this indentation correct?

Comment (malfet) [reply .github/workflows/auto_request_review.yml:13]: Fixed, but hopefully CI would have caught that :P

**Review by izaitsevfb — APPROVED** [review]

File: .github/workflows/auto_request_review.yml [hunk 2]
```diff
@@ -3,11 +3,14 @@ name: Auto Request Review
 on:
   pull_request:
     types: [opened, ready_for_review, reopened]
-
+
```
Comment (seemethere) [inline .github/workflows/auto_request_review.yml:6]:
```suggestion

```

Lint might fail here due to trailing whitespace

**Review by atalman — APPROVED** [review]

**Pull request closed without merging**